1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,28 @@
 function readingList(books) {
-  // Write your code here...
+  books.forEach((book, index) => {
+    let { title, author, alreadyRead, bookCoverImage } = book;
+    // create p tag
+    const p = document.createElement("p");
+    // Append content to p
+    p.appendChild(document.createTextNode(`${title} by ${author}`));
+    // Create li tag
+    const li = document.createElement("li");
+    if (index === 0) {
```

Comment:
Can it just be `listItemEl.style.backgroundColor = (book.alreadyRead) ? 'green' :'red'` ?

Reply:
You are right. I made a mistake. I thought  the first element of list  should be different colour. I didn't read the question properly.